client/js/widgets/widget.js

```diff
@@ -1833,8 +1834,22 @@ export class Widget extends StateManaged {
   async move(coordGlobal, localAnchor) {
     const coordParent = this.coordParentFromCoordGlobal(coordGlobal);
     const offset = getOffset(this.coordParentFromCoordLocal(localAnchor), {x: this.get('x'), y: this.get('y')})
-    const newX = Math.round(coordParent.x + offset.x);
-    const newY = Math.round(coordParent.y + offset.y);
+    let newX = Math.round(coordParent.x + offset.x);
+    let newY = Math.round(coordParent.y + offset.y);
+    const limit = this.get('moveLimit');
```

Comment:
maybe you should add a comment here something like

//Makes drag respect moveLimit property